jobs/mysql/templates/drain_user_setup.sql.erb

```diff
@@ -0,0 +1,5 @@
+<%
+  node_ip = spec.ip
+  hosts = (node_ip.split(".")[0..-2] + ["%"]).join('.')
```

Comment:
This is not the way we should grant the `drain` user.

This script will be run anytime the MariaDB nodes IPs are changing anyway, and you cannot assume that the nodes IPs will be following a specific addressing scheme. So, you could to loop over the list of Galera nodes here (using BOSH link) and output one `GRANT` per IP address.

But there is even simpler: this script only need to allow the current node to connect, i.e. `<%= spec.ip %>`, because similar `drain` script will exist on other nodes and take care for the proper GRANT, just in time it is necessary.

Reply:
> So, you could to loop over the list of Galera nodes here (using BOSH link) and output one GRANT per IP address
- Unfortunately, PostStartSQLFiles, cannot accept multiple command in one file, https://github.com/iamejboy/cf-mysql-release/blob/a51ea90cfe51318ba6f11e5d3497527c9e0dd5a3/jobs/mysql/templates/mariadb_ctl_config.yml.erb#L42 , 
so I resorted to like that as a work around to allow all nodes. This might need a change? https://github.com/cloudfoundry/mariadb_ctrl/blob/master/mariadb_helper/mariadb_helper.go#L257

> this script only need to allow the current node to connect, i.e. <%= spec.ip %>, because similar drain script will exist on other nodes and take care for the proper GRANT, just in time it is necessary.
- the script need to allow also other node ip, as when during drain on a local node, it will jump in to other node db to check its status.